### DISCUSSION TITLE
D.3.2. Collaboration - EKG/Maturity

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#85] [Page Comments]
# D.3.2. Collaboration - EKG/Maturity

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity.ekgf.org/pillar/technology/user-interface/collaboration/

**Comment by jgeluk** [comment]
This is about making sure that every object in the EKG has its own page (that's shown in a common "terminal" when a user clicks on an "EKG/IRI") that's linked to all means of communication such as Slack, Twitter, Google Drive, anything that's relevant to that given object.

**Reply by jgeluk** [reply]
Not all "collaboration" is very well structured (using predefined workflows for example), there usually is also a lot of more or disorganized collaboration around a given topic/object via systems like Slack, Discord, Zoom, and so forth. If the generic model-driven "terminal" to the EKG would support links from any given object in the EKG to any other communication channel then things would at least be a bit more organized and streamlined and "discoverable" via that terminal.
